MoreCyclopsUpgrades/Managers/PowerIconState.cs

```diff
@@ -1,34 +1,70 @@
 ﻿namespace MoreCyclopsUpgrades.Managers
 {
-    using System.Collections.Generic;
     using MoreCyclopsUpgrades.Modules;
+    using System.Collections.Generic;
 
     internal class PowerIconState
     {
+        internal struct PowerIcon
+        {
+            public TechType TechType;
+            public float Value;
+            public char Format;
```

Comment:
Replace this with a format string.  
Maybe even create an enum to categorize the various formats for numbers displayed aboard the cyclops.